src/app/page.tsx

```diff
@@ -97,7 +97,13 @@ const JurisdictionSection = () => {
 };
 
 const InvitationCta = () => (
-  <section className="usa-graphic-list usa-section usa-section--light-blue">
+  <section
+    className={classNames(
+      styles.invitationCtaSection,
+      'hide-footer-bar',
+      'usa-graphic-list usa-section usa-section--light-blue',
```

Comment:
We should be able to remove this extra class since we're using the image now.
```suggestion
      'usa-graphic-list usa-section',
```

Reply:
still need usa-section, but yes